sig-storage/charter.md

```diff
@@ -0,0 +1,62 @@
+# SIG Storage Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Storage is responsible for ensuring that different types of file and block storage
+(whether ephemeral or persistent, local or remote) are available wherever a container is
+scheduled (including provisioning/creating, attaching, mounting, unmounting, detaching,
+and deleting of volumes), storage capacity management (container ephemeral storage
+usage, volume resizing, etc.), influencing scheduling of containers based on storage
+(data gravity, availability, etc.), and generic operations on storage (snapshoting, etc.).
+
+### In scope
+
+Some notable examples of features owned by SIG Storage:
+
+* Persistent Volume Claims and Persistent Volumes
+* Storage Classes and Dynamic Provisioning
+* Kubernetes volume plugins
+* Container Storage Interface (CSI)
+* Secret, ConfigMap, DownwardAPI Volumes
```

Comment:
Don't know if this should be more explicit that its Secrets, ConfigMap etc related to Volumes?  I know that's pretty implicit due to the doc and such but just a nit.

Reply:
Do we want to mention enptydir volumes too?

Reply:
Done!